docs/docs/babel.md

```diff
@@ -2,19 +2,20 @@
 title: Babel
 ---
 
-Gatsby uses the phenomenal project [Babel](https://babeljs.io/) to enable support for writing modern JavaScript — while still supporting older browsers.
 
-## How to specify which browsers to support
+जुन्या ब्राऊजरला समर्थन देत असताना- गॅटस्बी (babel](https://babeljs.io/) आधुनिक जावास्क्रिप्ट लिहिण्यास समर्थन देण्यासाठी वापरतो 
 
-Gatsby supports by default the last two versions of major browsers, IE 9+, as well as any browser that still has 1%+ browser share.
+## कोणत्या ब्राउझरचे समर्थन करावे हे निर्दिष्ट कसे करावे
 
-This means that your JavaScript is automatically compiled to ensure it works on older browsers. Polyfills are also automatically added — no more shipping code which mysteriously breaks on older browsers!
+गॅटस्बी मुख्य ब्राउझरच्या शेवटच्या दोन आवृत्त्यांना डीफॉल्टनुसार समर्थन देते, IE 9+, तसेच कोणताही ब्राउझर ज्यात अद्याप 1% + ब्राउझर सामायिक आहे.
 
-If you only target newer browsers, see the [Browser Support](/docs/browser-support/) docs page for how to instruct Gatsby on which browsers you support and then Babel will start compiling for only these browsers.
+याचा अर्थ असा आहे की आपली जावास्क्रिप्ट जुन्या ब्राउझरवर कार्य करते हे सुनिश्चित करण्यासाठी स्वयंचलितपणे संकलित केली आहे. पॉलीफिल देखील स्वयंचलितपणे जोडली जातात - जुन्या ब्राउझरवर रहस्यमयपणे खंडित करणारा कोणताही शिपिंग कोड नाही!
 
-## How to use a custom .babelrc file
+आपण केवळ नवीन ब्राउझरला लक्ष्य केले असल्यास, आपण कोणत्या ब्राउझरवर समर्थित आहात यावर गॅटस्बीला कसे शिकवायचे यासाठी [ब्राउझर समर्थन] (/ दस्तऐवज / ब्राउझर-समर्थन /) दस्तऐवज पृष्ठ पहा आणि नंतर बाबेल केवळ या ब्राउझरसाठी कंपाईल करणे प्रारंभ करेल.
```

Comment:
dont try to translate the english words from links, links needs to be lin English for perfect work